### PR TITLE
added stakecapital

### PR DESCRIPTION
**Pull request by Loukaci**
Added logo and delegation informations

### CODE REVIEW

**Comment by aaronmgdr** [comment]
looks good but can you verify that that you are in fact associated with Stake Capital? 

your gh is new and the address provided is empty 

for example a twett from the staked capital account with a link to this pr.

**Comment by aaronmgdr** [comment]
run `yarn prettier`

**Comment by Loukaci** [comment]
hey @aaronmgdr  here is a tweet to confirm it.  

https://x.com/StakeCapital/status/1822974769915970021

**Comment by aaronmgdr** [comment]
tweet is great

unfortunatly you will have to handle the merge conflicts now

**Comment by Loukaci** [comment]
@aaronmgdr  Conflict has resolved .

**Review by aaronmgdr — DISMISSED** [review]

**Review by aaronmgdr — DISMISSED** [review]

**Review by Loukaci — COMMENTED** [review]
Ok for me.

**Review by Loukaci — COMMENTED** [review]
Ok for me

**Review by aaronmgdr — APPROVED** [review]

File: src/config/delegates.json [hunk 1]
```diff
@@ -185,7 +238,7 @@
       "website": "https://thepassivetrust.com",
       "twitter": "https://twitter.com/0xTPT"
     },
-    "interests": ["Privacy", "Prosperity", "Change"],
+    "interests": [ "Privacy",   "Prosperity",      "Change"    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:241]:
```suggestion
    "interests": ["Privacy", "Prosperity", "Change"],
```

File: src/config/delegates.json [hunk 2]
```diff
@@ -196,7 +249,7 @@
     "links": {
       "website": "https://plusv.io"
     },
-    "interests": ["Staking", "Security", "Passive Income"],
+    "interests": [ "Staking",  "Security",  "Passive Income"],
```
Comment (aaronmgdr) [inline src/config/delegates.json:252]:
```suggestion
    "interests": ["Staking", "Security", "Passive Income"],
```

File: src/config/delegates.json [hunk 3]
```diff
@@ -173,7 +222,11 @@
       "website": "https://www.easy2stake.com",
       "twitter": "https://twitter.com/easy2stake"
     },
-    "interests": ["Proof-of-Stake", "High Availability", "Infrastructure"],
```
Comment (aaronmgdr) [inline src/config/delegates.json:176]:
```suggestion
    "interests": ["Proof-of-Stake", "High Availability", "Infrastructure"],
```

File: src/config/delegates.json [hunk 4]
```diff
@@ -149,7 +191,11 @@
       "website": "https://virtualhive.io",
       "twitter": "https://twitter.com/VirtualHiveio"
     },
-    "interests": ["Web3 Infrastructure", "Reliability", "Sustainability"],
+    "interests": [
+      "Web3 Infrastructure",
+      "Reliability",
+      "Sustainability"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:198]:
```suggestion
    "interests": ["Web3 Infrastructure", "Reliability", "Sustainability"],
```

File: src/config/delegates.json [hunk 5]
```diff
@@ -161,7 +207,10 @@
       "website": "https://www.validator.capital",
       "twitter": "https://twitter.com/ValidatorCap"
     },
-    "interests": ["Web3 Investment", "De-Fi"],
+    "interests": [
+      "Web3 Investment",
+      "De-Fi"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:213]:
```suggestion
    "interests": ["Web3 Investment", "De-Fi"],
```

File: src/config/delegates.json [hunk 6]
```diff
@@ -137,7 +175,11 @@
       "website": "https://www.happycelo.com",
       "twitter": "https://twitter.com/HappyCelo_com"
     },
-    "interests": ["Security", "High Availability", "Monitoring"],
+    "interests": [
+      "Security",
+      "High Availability",
+      "Monitoring"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:182]:
```suggestion
    "interests": ["Security", "High Availability", "Monitoring"],
```

File: src/config/delegates.json [hunk 7]
```diff
@@ -30,7 +42,7 @@
     "links": {
       "twitter": "https://twitter.com/_iamoracle"
     },
-    "interests": ["Community", "Transparency", "Accountability", "Good Vibes"],
+    "interests": [ "Community", "Transparency", "Accountability", "Good Vibes" ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:45]:
```suggestion
    "interests": ["Community", "Transparency", "Accountability", "Good Vibes"],
```

File: src/config/delegates.json [hunk 8]
```diff
@@ -54,7 +66,13 @@
...
       "twitter": "https://twitter.com/reficolombia"
     },
-    "interests": ["ReFi", "Community", "Transparency", "Accountability", "Good Vibes"],
+    "interests": [
+      "ReFi",
+      "Community",
+      "Transparency",
+      "Accountability",
+      "Good Vibes"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:75]:
```suggestion
    "interests": ["ReFi", "Community", "Transparency", "Accountability", "Good Vibes"],
```

File: src/config/delegates.json [hunk 9]
```diff
@@ -42,7 +54,7 @@
       "website": "https://linktr.ee/0xj4an_work",
       "twitter": "https://twitter.com/0xj4an"
     },
-    "interests": ["Community", "Transparency", "Accountability", "Good Vibes"],
+    "interests": [ "Community", "Transparency", "Accountability", "Good Vibes" ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:57]:
```
    "interests": ["Community", "Transparency", "Accountability", "Good Vibes"],
```

File: src/config/delegates.json [hunk 10]
```diff
@@ -77,7 +89,10 @@
       "website": "https://staking.fund",
       "twitter": "https://twitter.com/stakingfund"
     },
-    "interests": ["Proof-of-Stake", "Infrastructure"],
+    "interests": [
+      "Proof-of-Stake",
+      "Infrastructure"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:95]:
```suggestion
    "interests": ["Proof-of-Stake", "Infrastructure"],
```

File: src/config/delegates.json [hunk 11]
```diff
@@ -101,7 +120,10 @@
       "website": "https://tessellated.io",
       "twitter": "https://twitter.com/tessellated_io"
     },
-    "interests": ["Proof-of-Stake", "Infrastructure"],
+    "interests": [
+      "Proof-of-Stake",
+      "Infrastructure"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:126]:
```suggestion
    "interests": ["Proof-of-Stake", "Infrastructure"],
```

File: src/config/delegates.json [hunk 12]
```diff
@@ -113,7 +135,10 @@
       "website": "https://moonli.me",
       "twitter": "https://twitter.com/moonli_me"
     },
-    "interests": ["Staking", "Decentralization"],
+    "interests": [
+      "Staking",
+      "Decentralization"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:141]:
```suggestion
    "interests": ["Staking", "Decentralization"],
```

File: src/config/delegates.json [hunk 13]
```diff
@@ -124,7 +149,10 @@
     "links": {
       "website": "https://www.usopp.club"
     },
-    "interests": ["Social Good", "Reliability"],
+    "interests": [
+      "Social Good",
+      "Reliability"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:155]:
```suggestion
    "interests": ["Social Good", "Reliability"],
```

File: src/config/delegates.json [hunk 14]
```diff
@@ -136,7 +164,11 @@
       "website": "https://www.swiftstaking.com",
       "twitter": "https://twitter.com/swiftstaking"
     },
-    "interests": ["The Graph", "Blockchain Infrastructure", "Trust"],
+    "interests": [
+      "The Graph",
+      "Blockchain Infrastructure",
+      "Trust"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:171]:
```suggestion
    "interests": ["The Graph", "Blockchain Infrastructure", "Trust"],
```

File: src/config/delegates.json [hunk 15]
```diff
@@ -184,7 +216,11 @@
       "website": "https://www.easy2stake.com",
       "twitter": "https://twitter.com/easy2stake"
     },
-    "interests": ["Proof-of-Stake", "High Availability", "Infrastructure"],
+    "interests": [
+      "Proof-of-Stake",
+      "High Availability",
+      "Infrastructure"
+    ],
```
Comment (aaronmgdr) [inline src/config/delegates.json:223]:
```suggestion
    "interests": ["Proof-of-Stake", "High Availability", "Infrastructure"],
```

**Review by aaronmgdr — APPROVED** [review]

File: src/config/delegates.json [hunk 16]
```diff
@@ -1,4 +1,16 @@
...
+  "0x3c3Eb5C41720D555F81eD554e901edD0a6aF228C": {
+    "name": "Stake Capital",
+    "address": "0x3c3Eb5C41720D555F81eD554e901edD0a6aF228C",
+    "logoUri": "/logos/delegates/stakecapital.jpg",
+    "date": "2024-08-08",
+    "links": {
+      "website": "https://www.stake.capital/",
+      "twitter": "https://x.com/StakeCapital"
+    },
+    "interests": [ "Proof-of-Stake","High Availability","Infrastructure","Web3 Investment"],
```
Comment (aaronmgdr) [inline src/config/delegates.json:11]:
```suggestion
    "interests": ["Proof-of-Stake", "High Availability", "Infrastructure", "Web3 Investment"],
```

File: src/config/delegates.json [hunk 17]
```diff
@@ -1,4 +1,16 @@
...
+    "address": "0x3c3Eb5C41720D555F81eD554e901edD0a6aF228C",
+    "logoUri": "/logos/delegates/stakecapital.jpg",
+    "date": "2024-08-08",
+    "links": {
+      "website": "https://www.stake.capital/",
+      "twitter": "https://x.com/StakeCapital"
+    },
+    "interests": ["Proof-of-Stake", "High Availability", "Infrastructure", "Web3 Investment"],
+    "description": "Stake Capital is a trusted and experienced blockchain investment firm, actively contributing to the blockchain ecosystem since 2016. With expertise in venture capital, incubation, acceleration, and validation, we ensure robust participation in governance through our mainnet validation for top projects and our involvement in testnets for emerging infrastructures. Our commitment to excellence and transparency makes us a reliable partner to delegate your voting power, helping to secure and innovate within the Celo network."
+  },
```
Comment (aaronmgdr) [inline src/config/delegates.json:13]:
```suggestion
  },
```

File: src/config/delegates.json [hunk 18]
```diff
@@ -1,4 +1,16 @@
 {
+  "0x3c3Eb5C41720D555F81eD554e901edD0a6aF228C": {
+    "name": "Stake Capital",
+    "address": "0x3c3Eb5C41720D555F81eD554e901edD0a6aF228C",
+    "logoUri": "/logos/delegates/stakecapital.jpg",
```
Comment (aaronmgdr) [inline src/config/delegates.json:5]:
```suggestion
"logoUri": "/logos/delegatees/stakecapital.jpg",
```

File: src/config/delegates.json [hunk 19]
```diff
@@ -1,4 +1,16 @@
 {
+  "0x3c3Eb5C41720D555F81eD554e901edD0a6aF228C": {
+    "name": "Stake Capital",
+    "address": "0x3c3Eb5C41720D555F81eD554e901edD0a6aF228C",
+"logoUri": "/logos/delegatees/stakecapital.jpg",
```
Comment (aaronmgdr) [inline src/config/delegates.json:5]:
```suggestion
    "logoUri": "/logos/delegates/stakecapital.jpg",
```

File: src/config/delegates.json (hunk 18 above)
Comment (aaronmgdr) [inline src/config/delegates.json:5]:
```suggestion
    "logoUri": "/logos/delegatees/stakecapital.jpg",
```

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**